framework/include/interfaces/Coupleable.h

```diff
@@ -84,6 +84,15 @@ class Coupleable
     return _coupled_vector_moose_vars;
   }
 
+  /**
+   * Get the list of scalar coupled variables
+   * @return The list of scalar coupled variables
+   */
+  const std::vector<MooseVariableScalar *> & getCoupledScalarMooseVars() const
```

Comment:
![image](https://user-images.githubusercontent.com/3036960/46752650-057e3100-cc7b-11e8-9805-2bf6d3aed450.png)

https://github.com/idaholab/moose/blob/devel/framework/include/interfaces/ScalarCoupleable.h#L51

Reply:
Sorry, I didn't even know that `ScalarCoupleable` existed... I'll verify that this is all I need and then close the issue/PR if so.

Reply:
Yeah - I don't really love that we have that separate interface. It wasn't my design but you'll find everything you need for Scalar in the other interface. It might just mean that you have to inherit from it if the object you are extending doesn't already inherit from it.